src/p2f/src/CodeOwls.PowerShell/CodeOwls.PowerShell.Provider/Provider.cs

```diff
@@ -110,7 +140,9 @@ string NormalizeWhacks( string path )
                 return PSDriveInfo.Root + NormalizeWhacks(sub);
```

Comment:
Path.Combine will automatically assemble paths using the correct form of "/" or "\\"

Reply:
By looking into the logic of NormalizeWhacks() in P2F, I do not think we need this function, thus commented all.  I am not certain about the jim's original design idea. So I am keeping the empty function in case later on we do need normalize it.